tutor/src/models/course/scores/task-result.js

```diff
@@ -124,58 +119,16 @@ export default class TaskResult extends BaseModel {
     return this.correct_on_time_exercise_count + this.correct_accepted_late_exercise_count;
   }
 
-  adjustAverages(prevCompletedStepCount, prevCorrect) {
-    const oldScore = this.score;
-    const period = this.student.period;
-    const { course } = period.coursePeriod;
-    const { type } = this;
-    let student = this.student;
+  @action adjustScore() {
+    this.is_included_in_averages = true;
```

Comment:
Not sure if this is correct. Only tasks that are past-due, have at least 1 exercise and are reading or homework should be in the averages.